quartz/i18n/index.ts

```diff
@@ -16,7 +16,6 @@ import vi from "./locales/vi-VN"
 import pt from "./locales/pt-BR"
 import hu from "./locales/hu-HU"
 import fa from "./locales/fa-IR"
-import pl from "./locales/pl-PL"
```

Comment:
this resolved poorly, can you make sure this is included again?

Reply:
Sorry - didn't spot this! Fixed now